### DISCUSSION TITLE
EZPF mods submitting scores with misses

### DISCUSSION THREAD

**Discussion by gitphfish** [ppy/osu-framework#6544] [Ideas]
I played C Type using EZPF, and submitted a score with a 100. I actually got a medal from it.

I played again a couple times and the EZPF mods were pretty inconsistent when it came to restarting. Some notes I could consistently miss or get 50's on, That includes both slider ends and full misses. I submitted another score with a miss shortly after.

The PF mod on its own seems fine, but I've tried EZPF on other maps and it seems consistent, though I've yet to check with another install. 

![Screenshot_20241211_183221](https://github.com/user-attachments/assets/6138a3a5-ff6e-4f40-9ece-5ab3074ab2e7)
![Screenshot_20241211_183225](https://github.com/user-attachments/assets/b1300713-d160-46f0-98be-fbcca9b80e9a)
![Screenshot_20241211_183836](https://github.com/user-attachments/assets/e4f5600f-2b30-49fa-92d5-5286199bec79)

Arch Linux, flatpak, 2024.1208.0

**Comment by bdach** [comment]
Wrong repository, and also duplicate (https://github.com/ppy/osu/issues/20511)
